charts/argo-workflows/values.yaml

```diff
@@ -157,7 +157,7 @@ controller:
 
   # -- Specifies the container runtime interface to use (one of: `docker`, `kubelet`, `k8sapi`, `pns`, `emissary`)
   ## Ref: https://argoproj.github.io/argo-workflows/workflow-executors/
-  containerRuntimeExecutor: docker
+  containerRuntimeExecutor: emissary
```

Comment:
Are you sure that this is needed?
> Docker (docker)¶
> ⚠️Deprecated.
> default in <= v3.2

> Emissary (emissary)¶
> alpha
> 
> v3.1 and after
> default in >= v3.3

Source: https://argoproj.github.io/argo-workflows/workflow-executors/#emissary-emissary

Reply:
Sorry, I mistakenly think I, this is the default configuration of 3.2.3